recipes/h5py/run_test.py

```diff
@@ -0,0 +1,24 @@
+import h5py
+import h5py._conv
+import h5py._errors
+import h5py._objects
+import h5py._proxy
+import h5py.defs
+import h5py.h5
+import h5py.h5a
+import h5py.h5d
+import h5py.h5f
+import h5py.h5fd
+import h5py.h5g
+import h5py.h5i
+import h5py.h5l
+import h5py.h5o
+import h5py.h5p
+import h5py.h5r
+import h5py.h5s
+import h5py.h5t
+import h5py.h5z
+import h5py.utils
+
+from sys import exit
+exit(0) if h5py.run_tests().wasSuccessful() else exit(1)
```

Comment:
Runs the full test suite. Errors if there was a failure. The suite is pretty fast so no point in not running it.